src/user.controller.ts

```diff
@@ -118,29 +120,127 @@ export class UserController {
         // Create a Paginated query to return all the users
         const allUsers: [User] = await this.repository
             .createQueryBuilder('user')
-            .select(['user.id', 'user.firstName', 'user.lastName', 'user.email'])
+            .select(['user.id', 'user.firstName', 'user.lastName', 'user.email', 'user.enabled'])
             .skip(request.offset)
             .take(request.limit)
             .where('user.email like :email and user.firstName like :firstName and user.lastName like :lastName', filter)
```

Comment:
Maybe we should include enabled as a parameter to search for here.

Reply:
Yes, i forgot that. I put only that parameter in the select just for test at first.